### PR TITLE
add vanilla jupyterbook to show features

### PR DESCRIPTION
**Pull request by isabelizimm**
In this PR, I created basic features and documentation for a vanilla jupyterbook setup using project Meteor.

To view the Jupyter Book as a webpage -> https://isabelizimm.github.io/meteor-demo/README.html

### CODE REVIEW

**Comment by durandom** [comment]
@MichaelClifford could you review please?

**Review by durandom — APPROVED** [review]
LGTM

**Comment by MichaelClifford** [comment]
Sure @durandom I can do a post-merge review : ) and @isabelizimm can added any changes into a subsequent PR.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
-project-template
+Meteor Demo
 ==============================
 
-template for the team to use
+This repository has materials for a hands-on tutorial to demonstrate tools developed by Red Hat's Artificial Intelligence Center of Excellence.
+
+This tutorial highlights the capabilities of [Project Meteor](https://github.com/AICoE/meteor), which is a web application that converts users' GitHub repositories into [JupyterBook](https://jupyterbook.org/intro.html) environments.
```
Comment (MichaelClifford) [inline README.md:6]: "... web application that converts users' GitHub repositories into both a [JupyterBook](https://jupyterbook.org/intro.html) as well as a deployment interactive environment.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+
+If you would like to suggest a new feature or report a bug, please open a new [issue](https://github.com/aicoe-aiops/meteor-demo/issues/new/choose).
+
+
+## Contact
+
+This project is maintained as part of Red Hat's Artificial Intelligence Center of Excellence. More AICoE projects can be found [here](https://github.com/AICoE).
+
 
 Project Organization
```
Comment (MichaelClifford) [inline README.md:24]: You can remove all this stuff below

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+title: Meteor Demo
+author: AIOps team
+# logo: logo.png
```
Comment (MichaelClifford) [inline _config.yml:3]: Can we add our logo here? 
image

File: _config.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+# See https://jupyterbook.org/customize/config.html#add-a-link-to-your-repository
+html:
+  use_issues_button: true
+  use_repository_button: true
+  google_analytics_id: ""  # A GA id that can be used to track book views.
+
+# Launch button settings
+launch_buttons:
+  notebook_interface        : jupyterlab  # The interface interactive links will activate ["classic", "jupyterlab"]
+  jupyterhub_url            : https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud  # The URL of the JupyterHub (e.g., https://datahub.berkeley.edu)
```
Comment (MichaelClifford) [inline _config.yml:29]: This needs to be updated to reflect the smaug cluster. Do we know if `https://jupyterhub-opf-jupyterhub.apps.smaug.massopen.cloud` will be the name? Can we inject env variables here somehow to maintain this repo without updating the hand coded the jupyter_url if we want to switch clusters?

File: _config.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+# Only build files specified by the _toc
+only_build_toc_files: true
+
+# Exclude any additional file types being built
+exclude_patterns: [.github/*, manifests/*,  CHANGELOG.md, docs/images/*]
+
+# Information about where the book exists on the web
+repository:
+  url: https://github.com/aicoe-aiops/meteor  # Online location of your book
```
Comment (MichaelClifford) [inline _config.yml:15]: https://github.com/aicoe-aiops/meteor-demo

File: _config.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+# Add GitHub buttons to your book
+# See https://jupyterbook.org/customize/config.html#add-a-link-to-your-repository
+html:
+  use_issues_button: true
+  use_repository_button: true
+  google_analytics_id: ""  # A GA id that can be used to track book views.
+
+# Launch button settings
+launch_buttons:
```
Comment (MichaelClifford) [inline _config.yml:27]: not sure if the deployed version is old but it looks like there is still a "Binder" deploy button. It should be removed if not already.

File: docs/community-native.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Community Native
+
+Jupyter Books are built-in with features to encourage contributing. Every page has a Github icon in the top corner, where users are offered prompts to suggest edits, open issues, and visit the source repository. This allows the beauty of open source to help raise the quality of your content.
```
Comment (MichaelClifford) [inline docs/community-native.md:3]: "Jupyter Books include built-in with features..."

File: docs/interact-with-code.ipynb [hunk 8]
```diff
@@ -0,0 +1,130 @@
+{
```
Comment (MichaelClifford) [inline docs/interact-with-code.ipynb:1]: you can't really interact with code here. Maybe a better title for the notebook and section would be "View Rendered Notebooks"

File: docs/rendering-text.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+# Building documentation
+
+Jupyter Books natively render **.rst, .md, and .ipynb** files natively from a single GitHub repo in a beautiful way.
```
Comment (MichaelClifford) [inline docs/rendering-text.md:3]: Jupyter Books natively render **.rst, .md, and .ipynb** files from a single GitHub repo in a beautiful way.

-removed "naively" twice

**Review by MichaelClifford — COMMENTED** [review]
Apologies for the post-merge review @isabelizimm

**Pull request merged**